source/simplify.d.ts

```diff
@@ -0,0 +1,4 @@
+/**
+Flatten the type output for ease of reference.
```

Comment:
"ease of reference"? I think it could more clearly state that it's to improve the type hints shown in editors.